Vhm aws permissions #14391 keichwa (#1198)
uyuni-project/uyuni-docs: modules/client-configuration/pages/vhm-aws.adoc

----
I1234567890abcdef0
----

== AWS Permissions for Virtual Host Manager

For security reasons, always grant the least privilege possible for a task to be performed.

> Thus using the Access Key for a user with excessive permissions to AWS is not advised.

Reviewer: "Using an Access Key with excessive permissions for users connecting to AWS is not advised."

Author: ```suggestion
Using an Access Key with excessive permissions for users connecting to AWS is not advised.
```
